public/app/plugins/panel/bargauge/BarGaugePanel.tsx

```diff
@@ -88,3 +89,17 @@ export class BarGaugePanel extends PureComponent<PanelProps<BarGaugeOptions>> {
     );
   }
 }
+
+const getDisplay = (view: DataFrameView, colIndex: number): DisplayProcessor => {
```

Comment:
actually looking some more....  what do you think about adding:

`getFieldDisplayProcessor(colIndex)` to DataFrameView?

then it could be reused for the other similar cases, and could be called above on line 40 as:
```
display={view?.getFieldDisplayProcessor(colIndex)}
```

Reply:
Yeah, totally agree! Much better! Will do the change.